indico/modules/events/persons/blueprint.py

```diff
@@ -25,4 +26,6 @@
 
 # EventPerson operations
 _bp.add_url_rule('/persons/<int:person_id>/edit', 'edit_person', RHEditEventPerson, methods=('GET', 'POST'))
+_bp.add_url_rule('/persons/<int:person_id>/unused/delete', 'delete_unused_person', RHDeleteUnusedEventPerson,
```

Comment:
I'd remove the `/unused/delete` part - it'd fit if this was a POST request, but for DELETE you usually point to the resource without including a verb/action in the url as well.